frontend/src/components/app/app.ts

```diff
@@ -24,66 +30,103 @@ export const getLocalStorage = () => {
   };
 };
 
-export const updateLocalStorage = (gameId: string, playerId: string, username: string) => {
+export const updateLocalStorage = (
+  gameId: string,
+  playerId: string,
+  username: string
+) => {
   localStorage.setItem('username', username);
   localStorage.setItem('gameId', gameId);
   localStorage.setItem('playerId', playerId);
 };
 
+export const clearLocalStorage = () => {
+  localStorage.removeItem('username');
+  localStorage.removeItem('gameId');
+  localStorage.removeItem('playerId');
+};
+
 @customElement('apollo-app')
 export class ApolloApp extends LitElement {
   static readonly is = 'apollo-app';
 
   static readonly styles = [shared, style];
 
-  gameConfigQuery = new ApolloQueryController(this, GameConfigQuery);
-  connectionRequestMutation = new ApolloMutationController(this, ConnectionRequestMutation);
-  gameConfigSubscription = new ApolloSubscriptionController(this, GameConfigUpdated);
+  @state()
+  protected _game: Game;
+
+  gameQuery = new ApolloQueryController(this, GameQuery);
 
-  private _connectionRequestInterval: typeof setInterval;
+  connectionRequestMutation = new ApolloMutationController(
+    this,
+    ConnectionRequestMutation
+  );
 
-  public async sendConnectionRequest(): void {
-    let storedGameConfig = {};
+  gameSubscription = new ApolloSubscriptionController(this, GameUpdated, {
+    noAutoSubscribe: true,
+    onData: ({ subscriptionData: { data } }) => {
+      this._game = data.game;
```

Comment:
FYI: `ApolloSubscriptionController` will update the host whenever data arrives. so you could just as easily `const game = this.gameSubscription.data.game` in `render()`, without having to first alias to `this._game`